cmd/releasego/check-limits.go

```diff
@@ -21,14 +22,14 @@ func init() {
 }
 
 func handleCheckLimits(p subcmd.ParseFunc) error {
-	pat := githubutil.BindPATFlag()
+	gitHubAuthFlags := githubutil.BindGitHubAuthFlags()
 
 	if err := p(); err != nil {
 		return err
 	}
 
 	ctx := context.Background()
-	client, err := githubutil.NewClient(ctx, *pat)
+	client, err := gitcmd.NewClientFromFlags(gitHubAuthFlags, ctx)
```

Comment:
I think this diff shows nicely why it makes more sense for this to be a method on `gitHubAuthFlags`, or at least in `githubutil`.